src/store/place/index.ts

```diff
@@ -3,6 +3,7 @@ import { buffers } from 'redux-saga';
 import { createSlice } from 'utils/@reduxjs/toolkit'; // Importing from `utils` makes them more type-safe ✅
```

Comment:
this file seems have many conflict now...?

Reply:
Yes. But all the changed function is necessary. How might I resolve this?? :(